core/trino-spi/src/main/java/io/trino/spi/session/PropertyMetadata.java

```diff
@@ -145,20 +170,30 @@ public String toString()
         return "PropertyMetadata{" + name + "}";
     }
 
-    public static PropertyMetadata<Boolean> booleanProperty(String name, String description, Boolean defaultValue, boolean hidden)
```

Comment:
We should keep the existing methods as `@Deprecated` and remove them at a future time.